sympy/physics/control/tests/test_lti.py

```diff
@@ -277,17 +278,25 @@ def test_TransferFunction_functions():
     stable_tf = TransferFunction(s**2 + a0 - a1*p, q*s - 1, s)
     stable_tf_ = TransferFunction(s**2 + a0 - a1*p, q*s - t, s)
 
-    assert G1.is_stable() is False
-    assert G2.is_stable() is True
-    assert tf1.is_stable() is False   # as one pole is +ve, and the other is -ve.
-    assert expect2.is_stable() is False
-    assert expect1.is_stable() is True
-    assert stable_tf.is_stable() is True
-    assert stable_tf_.is_stable() is True
-    assert TF_.is_stable() is False
+    assert G1.is_stable() is false
+    assert G2.is_stable() is true
+    assert tf1.is_stable() is true
+    assert expect2.is_stable() is false
+    assert expect1.is_stable() is true
+    assert stable_tf.is_stable() is true
+    assert stable_tf_.is_stable() is true
+    assert TF_.is_stable() is false
```

Comment:
Why are these lower case? I thought Python boolean was upper case?

Reply:
Is this backwards incompatible? Is this sympy `false/true` equivalent to `True/False`?